### ISSUE TITLE
[v0.1.5 mc1.10.2] reloading assets makes textures go missing

### ISSUE THREAD

**Issue by RCXcrafter** [asiekierka/UnlimitedChiselWorks#9]
When you reload the assets with f3 + t all the blocks from this mod get missing textures.

**Comment by asiekierka** [comment]
The 1.10/1.11 versions are a hack, things like this may and probably will happen, sadly.

**Comment by RCXcrafter** [comment]
I've been looking at your code and I think the problem is located here: https://github.com/asiekierka/UnlimitedChiselWorks/blob/1.10/src/main/java/pl/asie/ucw/UCWFakeResourcePack.java#L195
I am by no means an expert on this, but it seems to me you're clearing your cache of generated textures and not generating new ones.
If that's the case, it would mean the 1.12 version also has this issue.

**Comment by asiekierka** [comment]
@RCXcrafter No, this is done *specifically* so that the new ones would be generated.

The 1.12 version handles textures completely differently; the 1.10/1.11 versions lack key features necessary for this, so I load textures using a hack. (1.10 LiteLoader already breaks that hack, so I wouldn't be surprised if other things do as well.)

**Comment by P3rf3ctXZer0** [comment]
I can confirm this issue is deep. Check this chisel post!
https://github.com/Chisel-Team/ConnectedTexturesMod/issues/9
